src/main/kotlin/uk/gov/dluhc/printapi/database/entity/PrintRequestStatus.kt

```diff
@@ -64,6 +64,20 @@ class PrintRequestStatus(
 
     @Override
     override fun toString(): String {
-        return this::class.simpleName + "(id = $id , dateCreated = $dateCreated , createdBy = $createdBy , version = $version )"
+        return this::class.simpleName + "(id = $id , status = $status , dateCreated = $dateCreated , createdBy = $createdBy , version = $version )"
```

Comment:
I've added the `status` field to the `toString` method in this class, and also `TemporaryCertificateStatus` as per a late suggestion in my last PR - FYI @abdullah-sanver-valtech

Reply:
Thank you, @NathanRussellValtech